platform-sdk/platform-apps/tests/AddressBookTestingTool/src/main/java/com/swirlds/demo/addressbook/AddressBookTestingToolState.java

```diff
@@ -644,14 +681,21 @@ private AddressBook parseAddressBook(@NonNull final String addressBookString) th
     @NonNull
     private String getLastAddressBookFileEndsWith(@NonNull final String suffix) throws IOException {
         final Path addressBookDirectory = Path.of(addressBookConfig.addressBookDirectory());
+        File[] files = addressBookDirectory.toFile().listFiles(File::isFile);
```

Comment:
It's ok if you don't change this since you've already got it working. But in general and in the future, we should try and use the newer `Path` APIs instead of the old `File` ones.

Reply:
I had a [prior implementation that was using `Files.list()` in a try with resources](https://github.com/hashgraph/hedera-services/blob/c94fc287e997dd84489dcd24f4fd1a6a5aa721c3/platform-sdk/platform-apps/tests/AddressBookTestingTool/src/main/java/com/swirlds/demo/addressbook/AddressBookTestingToolState.java#L648), but I was baffled why it was not listing all files.  I went with this because I could get it to work.   Leaving it in for now.